api/envoy/extensions/transport_sockets/tls/v3/tls_spiffe_validator_config.proto

```diff
@@ -57,4 +82,9 @@ message SPIFFECertValidatorConfig {
 
   // This field specifies trust domains used for validating incoming X.509-SVID(s).
   repeated TrustDomain trust_domains = 1 [(validate.rules).repeated = {min_items: 1}];
+
+  // This field specifies a trust domain mapping as a json object. Mutually
+  // excluse with trust_domains.
+  config.core.v3.DataSource trust_bundle_map = 2;
```

Comment:
Maybe name it `trust_bundles`. In case we want to provide non-map structure input?

Reply:
We could add a separate repeated field of trust_bundles if users wanted to specify a set of individual trust bundles instead, but AFAIK SPIFFE Trust Bundle Map is the pattern for multi-bundle handling.

Reply:
I didn't get it. Basically, my concern is to give the biggest flexibility to the input format in the future and don't want it be limited by the name. Simple `trust_bundles` means the input may contains multiple bundles, not means we must use different `repeated`.

Reply:
The way to specify multiple bundles was decided to be the [SPIFFE bundle map](https://docs.google.com/document/d/13KHycYIfRC-g42aEfo4_4inF_WauCXOBgZAcUGuxdgs/edit#heading=h.o2sg9lu1e74v).  I don't know why anyone would want to do something else.

If we were to do that we'd have to wrap the field and include an enum indicating the file format.  I don't foresee anyone needing to specify multiple trust bundles in an out-of-spec way, given how straightforward the SPIFFE bundle map is.